docs/sections/user_guide/uw_yaml/rocoto_yaml.rst

```diff
@@ -621,3 +624,5 @@ The XML will look like this
 
      </metatask>
    </metatask>
+
+* The optional attributes ``mode`` and ``throttle`` are accepted under an ``attrs:`` key. See the :rocoto:`Rocoto documentation<>` for details.
```

Comment:
Just wanted to double check that nothing is suppose to go between `Rocoto documentation<>`.  Possible I'm just not familiar with this syntax.